pkg/webui/components/header-v2/header-v2.styl

```diff
@@ -0,0 +1,33 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.container
+  padding: 0 $cs.m
+  background: $c['tts-primary-050']
+  z-index: $zi.nav
+  border-bottom: 1px solid $c-divider
+  display: flex
+  justify-content: space-between
+  align-items: center
+  height: 4rem
+
+.mobile-menu
+  display: flex
+  align-items: center
+  gap: $cs.xs
+  +media-query-min($bp.s)
+    display: none
```

Comment:
All of this can be done with utility classes:
`.d-flex .al-center .gap-cs-xs .s:d-none`